app/controllers/movies_controller.rb

```diff
@@ -1,41 +1,42 @@
 class MoviesController < ApplicationController
   def index
     if params[:top_40]
-      top_40
-    else 
-      keyword_search
+      fetch_movies('top')
+    else
+      fetch_movies('search')
     end
   end
 
-  def top_40
-    response_1 = conn.get("3/discover/movie?api_key=#{ENV['MOVIES_API_KEY']}&language=en-US&sort_by=vote_average.desc&include_adult=false&include_video=false&page=1")
-    
```

Comment:
@dhalverson Do you know why we originally included the params for include_adult = false and include_video = false? I can see why we exclude adult (lol) but what does excluding video do? 

Rubocop gave violations for this line being too long so I removed them because I wasn't sure if we needed them anyway. What do you think?

Reply:
Per our convo, we will exclude adult videos but will include videos (sports, concerts, etc.)